### PR TITLE
Add Hotels Page

### PR DESCRIPTION
**Pull request by CannonLock**
I noticed the direct link provided defaults to creating a reservation for the full week and not just the in person section so I made sure that was clear.

### CODE REVIEW

**Comment by osg-cat** [comment]
@CannonLock, could I get a version of this page that hews a bit more closely to the model from 2020?

**Comment by CannonLock** [comment]
@osg-cat Absolutely, take a look now.

File: docs/2022/hotels.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# OSG All-Hands Meeting 2022 - Hotels
+
+The [OSG](https://osg-htc.org) All-Hands Meeting 2020 will be held March 14-18, with the in-person section spanning March 14-16.
```
Comment (osg-cat) [inline docs/2022/hotels.md]: 2022

Comment (osg-cat) [reply docs/2022/hotels.md]: “in-person section” => “in-person component”

File: docs/2022/hotels.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# OSG All-Hands Meeting 2022 - Hotels
+
+The [OSG](https://osg-htc.org) All-Hands Meeting 2020 will be held March 14-18, with the in-person section spanning March 14-16. 
+The meeting with be hosted in the [Thurman J. White Forum Building](https://pacs.ou.edu/about/conference-services/location/)
```
Comment (osg-cat) [inline docs/2022/hotels.md]: “with” => “will”

File: docs/2022/hotels.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# OSG All-Hands Meeting 2022 - Hotels
+
+The [OSG](https://osg-htc.org) All-Hands Meeting 2020 will be held March 14-18, with the in-person section spanning March 14-16. 
+The meeting with be hosted in the [Thurman J. White Forum Building](https://pacs.ou.edu/about/conference-services/location/) 
+(1704 Asp Ave) at the [University of Oklahoma](https://www.ou.edu/).
+
+## Lodging Options
+
+### [LaQuinta](https://www.wyndhamhotels.com/laquinta/norman-oklahoma/la-quinta-oklahoma-city-norman/overview)
```
Comment (osg-cat) [inline docs/2022/hotels.md]: Name = “La Quinta Inn & Suites by Wyndham Oklahoma City Norman”

File: docs/2022/hotels.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+- Located about [3 miles from the conference venue](https://www.google.com/maps/dir/Thurman+J.+White+Forum+Building,+1704+Asp+Avenue,+Norman,+OK+73072/La+Quinta+Inn+%26+Suites+by+Wyndham+Oklahoma+City+Norman,+930+Ed+Noble+Dr,+Norman,+OK+73072/@35.194631,-97.4815439,14z/data=!3m1!4b1!4m14!4m13!1m5!1m1!1s0x87b2682eef70daf9:0xa545a18f3b614551!2m2!1d-97.4452463!2d35.1979906!1m5!1m1!1s0x87b269009c755e43:0x7d6828ef878198cf!2m2!1d-97.483177!2d35.2072214!3e0)
+- Reserve a room:
+    - [On their website](https://www.wyndhamhotels.com/laquinta/norman-oklahoma/la-quinta-oklahoma-city-norman/rooms-rates?&checkInDate=03/14/2022&checkOutDate=03/18/2022&groupCode=CGLKED)
+    - Call [405-579-4000](tel:405-579-4000) and use 'OSG All hands meeting' to get special rate
+- Conference rate: $80.00/night + taxes
+- Check in time is 3 p.m., check out time is noon
+
+### [Embassy Suites](https://www.hilton.com/en/hotels/okceses-embassy-suites-norman-hotel-and-conference-center/)
```
Comment (osg-cat) [inline docs/2022/hotels.md]: Name = “Embassy Suites by Hilton Norman Hotel & Conference Center”

File: docs/2022/hotels.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+- Located about [6 miles from the conference venue](https://www.google.com/maps/dir/Thurman+J.+White+Forum+Building,+1704+Asp+Ave,+Norman,+OK+73072/2501+Conference+Dr,+Norman,+OK+73069/@35.2136079,-97.4992128,13.06z/data=!4m14!4m13!1m5!1m1!1s0x87b2682eef70daf9:0xa545a18f3b614551!2m2!1d-97.4452463!2d35.1979906!1m5!1m1!1s0x87b269810e60a861:0xf23b5979b16c6a77!2m2!1d-97.4782273!2d35.2436275!3e0)
+- Reserve a room:
+    - On their website (Coming Soon)
+    - Call [405-364-8040](tel:405-364-8040) and use 'OSG All hands meeting' to get special rate
+- Conference rate: $129.00/night + taxes
+- Check in time is 4 p.m., check out time is noon
+
+### [Holiday Inn Express](https://www.ihg.com/holidayinnexpress/hotels/us/en/norman/okcnn/hoteldetail)
```
Comment (osg-cat) [inline docs/2022/hotels.md]: Per emails with Kristin, remove this one entirely.

**Review by osg-cat — CHANGES_REQUESTED** [review]
After these changes, I’ll merge.

**Comment by CannonLock** [comment]
@osg-cat

**Pull request merged**